### PR TITLE
V1.1.3

### PR DESCRIPTION
**Pull request by moorepants**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by moorepants** [comment]
Seems like the environment varible `$SP_DIR` is not properly aware of the Python version. This should point to python3.6:

```
mv: cannot move `/home/conda/feedstock_root/build_artifacts/skijumpdesign_1525450019672/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/static' to `/home/conda/feedstock_root/build_artifacts/skijumpdesign_1525450019672/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python2.7/site-packages/': No such file or directory
```

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,7 +13,12 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: |
+    python -m pip install --no-deps --ignore-installed .
+    # This is a hack to get the custom css file in the right place. Remove once
+    # https://gitlab.com/moorepants/skijumpdesign/issues/44 is addressed
+    # properly.
+    mv {{ environ["PREFIX"] }}/static {{ environ["SP_DIR"] }}/
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should be able to add it to `MANIFEST.in`.

**Comment by jakirkham** [comment]
FYI "Skip branches without `appveyor.yml`" was not checked in the AppVeyor settings for some reason.

cc @isuruf

File: recipe/meta.yaml (hunk 1 above)
Comment (moorepants) [reply recipe/meta.yaml]: It is in the MANIFEST.in. See this SO question for the details:

https://stackoverflow.com/questions/50160009/how-can-i-package-a-css-file-with-a-single-module-flask-application-with-setupto

**Comment by moorepants** [comment]
Merging this without the hack.

**Comment by jakirkham** [comment]
Does it work without the hack?

**Comment by moorepants** [comment]
It functions, just doesn't have the css assets loaded, so the web app doesn't look right.

**Pull request merged**